From 1a2ed09bea7457cbb98eac5a2677f90bc57742a3 Mon Sep 17 00:00:00 2001
From: Stephane Viau <sviau@codeaurora.org>
Date: Fri, 24 Apr 2015 11:38:05 -0400
Subject: [PATCH 14/44] clk/mdss: Keep MSM MDSS clock driver

MSM MDSS clock driver is located in driver/video/msm/mdss folder.
DRM/KMS driver still needs this in msm-3.18 kernel.

Change-Id: Ibb1ec859068418385920891a8485726d442858a2
Signed-off-by: Stephane Viau <sviau@codeaurora.org>
---
 drivers/video/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/video/Makefile b/drivers/video/Makefile
index 437a23f..95b761a 100644
--- a/drivers/video/Makefile
+++ b/drivers/video/Makefile
@@ -13,3 +13,5 @@ obj-$(CONFIG_VIDEOMODE_HELPERS) += display_timing.o videomode.o
 ifeq ($(CONFIG_OF),y)
 obj-$(CONFIG_VIDEOMODE_HELPERS) += of_display_timing.o of_videomode.o
 endif
+
+obj-$(CONFIG_MSM_MDSS_PLL)        += msm/mdss/mdss_io_util.o
-- 
2.8.2

